pipeline/scripts/update-latest-yml.js

```diff
@@ -6,6 +6,7 @@ const YAML = require('js-yaml');
 const path = require('path');
 
 const parentDir = process.argv[2];
+const platform = process.argv.length > 3 ? process.argv[3] : process.platform;
 
 const getLatestYAMLPath = parentDir => {
     const platformModifier = process.platform == 'darwin' ? '-mac' : process.platform == 'linux' ? '-linux' : '';
```

Comment:
Should we be replacing the other usages of `process.platform` in this file with the new parameter?